libs/chiliui/chili/controls/object.lua

```diff
@@ -705,6 +722,7 @@ end
 --// =============================================================================
 
 function Object:CallListeners(listeners, ...)
+	if listeners == nil then return end
```

Comment:
is this really faster then going 0 loops  ?

Reply:
This is needed cause now OnClick and friends (the listeners tables) arent empty allocations for every table